string info
uint64 time_stamp
uint16[3] distance
